scrapy_poet/cache.py

```diff
@@ -16,7 +16,8 @@ def __getitem__(self, fingerprint: str) -> Any:
     def __setitem__(self, fingerprint: str, value) -> None:
         pass
 
-    def close(self):
+    @abc.abstractmethod
```

Comment:
Why should it be an abstractmethod? Empty close seems like a good default, in case a backend doesn't need to close anything.

Reply:
Reverted it. https://github.com/scrapinghub/scrapy-poet/pull/89/commits/4a8807abbddc5f84f039a726ac22631d5b760d52